lib/ansible/modules/system/user.py

```diff
@@ -194,6 +194,15 @@
         type: bool
         default: 'no'
         version_added: "2.4"
+    uid_min:
+        description:
+            - Minimal UID
+        version_added: "2.7"
+    uid_max:
+        description:
+            - Max UID
+        version_added: "2.7"
```

Comment:
The documentation for these new options needs to explain on what platforms it is supported, that it only matters on user creation, and that for BSD, both are required.